components/CharacterCard/CharacterCard.js

```diff
@@ -1 +1,28 @@
-export function CharacterCard() {}
+export function CharacterCard() {
+  const card = document.createElement("li");
+
+  card.classList.add("card");
+  card.innerHTML = `<div class="card__image-container">
+            <img
+              class="card__image"
```

Comment:
src and alt should be dynamic

Reply:
@E-LeeSchaffer we add all the dynamic parts, check again please